ads/_a4a-config.js

```diff
@@ -84,3 +84,21 @@ export const signingServerURLs = {
   'cloudflare': 'https://amp.cloudflare.com/amp-ad-verifying-keyset.json',
   'cloudflare-dev': 'https://amp.cloudflare.com/amp-ad-verifying-keyset-dev.json',
 };
+
+/** @type {!../extensions/amp-a4a/0.1/refresh-manager.RefreshConfig} */
+const DEFAULT_REFRESH_CONFIG = {
```

Comment:
Why do we need this here?  Why don't we just pass the config into RefreshManager when its created within amp-ad-network-doubleclick-impl?